runtime/integration-tests/src/xcm/development/tests/connectors.rs

```diff
@@ -1088,8 +1089,81 @@ fn allow_pool_should_fail() {
 			DispatchError::Token(sp_runtime::TokenError::Unsupported)
 		);
 
-		// TODO(subsequent PR): Add noop test for InvalidDomain
-		// NOTE: Blocked by https://github.com/centrifuge/centrifuge-chain/pull/1393
+		// Should fail if currency is not connectors transferable
+		assert_ok!(OrmlAssetRegistry::update_asset(
+			RuntimeOrigin::root(),
+			pool_currency_id,
+			None,
+			None,
+			None,
+			None,
+			None,
+			Some(CustomMetadata {
+				transferability: CrossChainTransferability::None,
+				mintable: Default::default(),
+				permissioned: Default::default(),
+				pool_currency: true,
+			}),
+		));
+		assert_noop!(
+			Connectors::allow_pool_currency(
+				RuntimeOrigin::signed(BOB.into()),
+				pool_id,
+				pool_currency_id,
+			),
+			pallet_connectors::Error::<DevelopmentRuntime>::AssetNotConnectorsTransferable
+		);
+
+		// Should fail if currency does not have any MultiLocation in metadata
+		assert_ok!(OrmlAssetRegistry::update_asset(
+			RuntimeOrigin::root(),
+			pool_currency_id,
+			None,
+			None,
+			None,
+			None,
+			None,
+			Some(CustomMetadata {
```

Comment:
Am I not seeing this well or is this `update_asset` exactly the same as the one above?

Reply:
The update above that has `CrossChainTransferability::None` whereas this one has `CrossChainTransferability::Connectors`. I improved awareness of the changes in [5396f5f](https://github.com/centrifuge/centrifuge-chain/pull/1364/commits/5396f5fbd3a1995a6748c17f6cae3603473c154f) as this very difficult to follow due to the amount of entries required for an update.